certbot-apache/certbot_apache/constants.py

```diff
@@ -131,6 +131,16 @@
 MOD_SSL_CONF_DEST = "options-ssl-apache.conf"
 """Name of the mod_ssl config file as saved in `IConfig.config_dir`."""
 
+
+UPDATED_MOD_SSL_CONF_DIGEST = ".updated-options-ssl-apache-conf-digest.txt"
+"""Name of the hash of the updated or informed mod_ssl_conf as saved in `IConfig.config_dir`."""
+
+ALL_SSL_OPTIONS_HASHES = [
+    '2086bca02db48daf93468332543c60ac6acdb6f0b58c7bfdf578a5d47092f82a',
+    '4844d36c9a0f587172d9fa10f4f1c9518e3bcfa1947379f155e16a70a728c21a',
```

Comment:
Where did this hash come from?

Also, I think we're missing a couple hashes here:
```
$ cd $(git rev-parse --show-toplevel)
$ git checkout 291c5c71327640a02218d7adda3e1013cd701b71
Note: checking out '291c5c71327640a02218d7adda3e1013cd701b71'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may 
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 291c5c7... Remove nonsensical Apache config directives
$ sha256sum letsencrypt-apache/letsencrypt_apache/options-ssl-apache.conf 
5a922826719981c0a234b1fbcd495f3213e49d2519e845ea0748ba513044b65b  letsencrypt-apache/letsencrypt_apache/options-ssl-apache.conf
$ git checkout 120a94f84f3b9e33f5095b20cfe8b7925815a969
Previous HEAD position was 291c5c7... Remove nonsensical Apache config directives
HEAD is now at 120a94f... mv letsencrypt_* letsencrypt-*.
$ sha256sum letsencrypt-apache/letsencrypt_apache/options-ssl-apache.conf 
4066b90268c03c9ba0201068eaa39abbc02acf9558bb45a788b630eb85dadf27  letsencrypt-apache/letsencrypt_apache/options-ssl-apache.conf
$
```